src/APIBasedAuth.ts

```diff
@@ -373,6 +393,37 @@ declare namespace APIBasedAuth {
     session?: string;
     username?: string;
   };
+
+  export type InitiateSignupResponse = {
+    userConfirmed: boolean;
+  };
+
+  export type InitiateSignupData = {
+    clientId: string;
+    /** Unique username for the user. A required field */
+    username: string;
+    /** Unique email address for the user. A required field */
+    email: string;
+    /**
+     * Web url where the user registration takes place.
+     * @example 'https://example.com'
+     */
+    originalUrl?: string;
+    /**
+     * Password for the user. In the case that a password is not
+     * provided it is assumed that this is a passwordless user and
+     * a password is auto generated for them on the server. The
+     * user is still able to go through the forgot password flow
+     * to set it at a later time
+     */
+    password?: string;
+    /**
+     * A phone number for the user
+     */
+    phone?: string;
```

Comment:
Any idea what the format should be here? e.g. `+1 123 456 789`, or `+1123456789`, or maybe something different?

Reply:
@swain I should add this as a comment. but the schema json is `"pattern": "^\\+[0-9]{11}$"` so I think `+1123456789` is the example